integration_tests/fastapi_mount_test.sh

```diff
@@ -24,6 +24,5 @@ until curl --fail --silent "$URL" > /dev/null; do
 done
 
 
-curl --fail --silent "http://localhost:$PORT/lilac_sub/api/v1/tasks/"
-
-curl --fail --silent "http://localhost:$PORT/lilac_sub/"
+curl --fail --silent "http://localhost:$PORT/$MOUNT_PATH/api/v1/tasks/" > /dev/null
+curl --fail --silent "http://localhost:$PORT/$MOUNT_PATH/" --max-redirs 0 > /dev/null
```

Comment:
```suggestion
curl --fail --silent "http://localhost:$PORT/$MOUNT_PATH/" -L --max-redirs 0 > /dev/null
```

Need to add `-L` (follow redirects) for `--max-redirs` to have any effect

Reply:
Thanks! Done :)

Reply:
I had to migrate this to a python test, FYI.